include/seqan3/core/type_traits/basic.hpp

```diff
@@ -60,14 +61,14 @@ using remove_rvalue_reference_t = typename remove_rvalue_reference<t>::type;
 // is_constexpr_default_constructible
 // ----------------------------------------------------------------------------
 
-/*!\brief Whether a type std::is_default_constructible in `constexpr`-context [UnaryTypeTrait].
+/*!\brief Whether a type std::is_default_constructible in `constexpr`-context [TypeTrait].
  * \tparam t The type to operate on.
  */
 template <typename t>
 struct is_constexpr_default_constructible : std::false_type
 {};
 
-/*!\brief Whether a type std::is_default_constructible in `constexpr`-context [UnaryTypeTrait specialisation].
+/*!\brief Whether a type std::is_default_constructible in `constexpr`-context [TypeTrait specialisation].
```

Comment:
We have no `\implements` / concept for this?

Reply:
What exactly is a TypeTrait? There is no definition at cppreference: https://en.cppreference.com/w/cpp/named_req/TypeTrait.